source/_data/ui.yml

```diff
@@ -267,7 +267,7 @@
   ring: Hold
   isNew: true
   description: "[Jasmine](https://jasmine.github.io/) was one of the earliest Javascript testing tools and comes as a package with in-built assertion utilities."
-  opinion: Jasmine doesn't come with a test runner, while Mocha does. We've put Jasmine on hold, with Mocha + Chai being adopted.
+  opinion: Jasmine doesn't come with a test runner, while Mocha does. We've put Jasmine on hold, with Mocha + Chai being at adopt.
```

Comment:
@Vasanth-Indix Not sure if we need this change; the original flows better even as a standalone sentence.